### DISCUSSION TITLE
How to design progress bar?

### DISCUSSION THREAD

**Discussion by CryptainGolD** [telegraf/telegraf#2047] [Ideas]
I saw progress bar in many channels such as
![image](https://github.com/user-attachments/assets/ca8da300-96cb-4d32-91da-18353b65b8d0)
![image](https://github.com/user-attachments/assets/bd557b39-ba36-46ed-8c76-b705c35f5d67)
![image](https://github.com/user-attachments/assets/9102ad16-ba57-4f7a-905b-eb092add11d5)

So I copied that text, but the result on vs code is: 🐸10%🫥🫥🫥🫥🫥🫥🫥🫥🫥
Funny, 🤣 🤣 🤣 
Of course there is a way for this like: ████░░░░░░
but as you can see, what the fuck design!
How to implement it? Maybe related with MarkdownV2?

**Comment by paveldemko** [comment]
They are using custom emojis, in the first and third cases, the second case just Telegram poll default functionality.
You need to buy a name on a fragment and pay a one-time fee for using custom emojis for a bot (about 1000 TON = ~ $5000).
So, you can use just emojis like that 🟩🟩🟩🟩🟩🟩⬜️⬜️⬜️⬜️

**Reply by CryptainGolD** [reply]
I bought premium and be able to copy that feagure like as text
![image](https://github.com/user-attachments/assets/be253492-8765-4518-a62f-4b8978b7ad3a)

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

![image](https://github.com/user-attachments/assets/efeea2a4-a4d3-4347-ac03-7b132bae42e9)

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

![image](https://github.com/user-attachments/assets/679e7641-360b-4704-bc2f-f3a8d93767c6)

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

Isn't it depended on any parse_mode?

**Comment by paveldemko** [comment]
you bought premium and can use custom emoji from your account, but you can't send them from your bots.
[Telegram docs](https://core.telegram.org/bots/api#sendinvoice:~:text=emoji%20sticker.-,Custom%20emoji%20entities%20can%20only%20be%20used%20by%20bots%20that%20purchased%20additional%20usernames%20on%20Fragment.,-Markdown%20style)

**Reply by paveldemko** [reply]
then you need pay 1000 TON for assign some username with bot, and then you can use custom emoji in bot messages

**Reply by paveldemko** [reply]
<img width="745" alt="image" src="https://github.com/user-attachments/assets/7aa0967f-e53a-4d70-a58c-dfd61de62691">

**Reply by CryptainGolD** [reply]
Ah, make sense now. Thanks for you help 🙏
